yb-voyager/cmd/exportData.go

```diff
@@ -340,6 +360,54 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 	return nil
 }
 
+func getConnectionUriForDebezium(s srcdb.Source) (string, error) {
+	if s.DBType == "oracle" {
+		connectionStringRegex := regexp.MustCompile(`.*connectString="(?P<connectString>.*)".*`)
+		match := connectionStringRegex.FindStringSubmatch(s.Uri)
+		if match == nil || len(match) != 2 {
+			return "", fmt.Errorf("unexpected URI format. regex matches = %v", match)
+		}
+		connectionString := fmt.Sprintf("jdbc:oracle:thin:@%s", match[1])
+		return connectionString, nil
+	}
+	return s.Uri, nil
+}
+
+// oracle wallet location can be optionally set in $TNS_ADMIN/ojdbc.properties as
+// oracle.net.wallet_location=<>
+func isOracleJDBCWalletLocationSet(s srcdb.Source) (bool, error) {
+	if s.DBType != "oracle" {
+		return false, nil
+	}
+	tnsAdmin, err := getTNSAdmin(s)
+	if err != nil {
+		return false, fmt.Errorf("failed to get tns admin")
+	}
+	ojdbcPropertiesFilePath := filepath.Join(tnsAdmin, "ojdbc.properties")
+	if _, err := os.Stat(ojdbcPropertiesFilePath); errors.Is(err, os.ErrNotExist) {
+		// file does not exist
+		return false, nil
```

Comment:
why returning `nil` here? so that it picks up the default?

Reply:
Yes